tools/hcluster_sg_parser/hcluster_sg_parser.xml

```diff
@@ -4,26 +4,25 @@
 <![CDATA[
 perl $__tool_directory__/hcluster_sg_parser.pl
 $inputFile
-> $report
 ]]>
     </command>
     <inputs>
         <param name="inputFile" type="data" format="tabular" label="hcluster output file in 3-column format" help="3-columns format: cluster_id cluster-size cluster-members" />
     </inputs>
     <outputs>
-        <data format="txt" name="report" hidden="true">
-            <discover_datasets pattern="(?P&lt;designation&gt;.+)_output\.txt" visible="true" />
-        </data>
+        <collection name="ids_lists" type="list" label="${tool.name} on ${on_string}">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)_output\.txt" format="txt" />
+        </collection>
     </outputs>
     <tests>
         <test>
             <param name="inputFile" ftype="tabular" value="hcluster_sg.tabular" />
-            <output name="report" file="empty_file" hidden="true">
+            <output_collection name="ids_lists" type="list">
                 <discovered_dataset designation="0" file="0_output.txt" />
```

Comment:
Thanks @jmchilton! `planemo lint` and `planemo test` were not complaining, should I file an issue?

Also, even after fixing this, at the end of `planemo test` I get:

```
base.twilltestcase DEBUG 2015-08-18 10:37:45,903 keepoutdir: /tmp/tmpeqFwFT/jobfiles, ofn: /tmp/tmpeqFwFT/jobfiles/2_output.txt
base.twilltestcase DEBUG 2015-08-18 10:37:45,903 ## GALAXY_TEST_SAVE=/tmp/tmpeqFwFT/jobfiles. saved /tmp/tmpeqFwFT/jobfiles/2_output.txt
ok

----------------------------------------------------------------------
XML: /tmp/tmpeqFwFT/xunit.xml
----------------------------------------------------------------------
Ran 1 test in 10.822s

OK
functional_tests.py INFO 2015-08-18 10:37:45,904 Shutting down
functional_tests.py INFO 2015-08-18 10:37:45,904 Shutting down embedded web server
functional_tests.py INFO 2015-08-18 10:37:45,906 Embedded web server stopped
functional_tests.py INFO 2015-08-18 10:37:45,906 Shutting down app
galaxy.jobs.handler INFO 2015-08-18 10:37:45,906 sending stop signal to worker thread
galaxy.jobs.handler INFO 2015-08-18 10:37:45,906 job handler queue stopped
galaxy.jobs.runners INFO 2015-08-18 10:37:45,906 LWRRunner: Sending stop signal to monitor thread
galaxy.jobs.runners INFO 2015-08-18 10:37:45,906 LWRRunner: Sending stop signal to 3 worker threads
galaxy.jobs.runners INFO 2015-08-18 10:37:45,907 TaskRunner: Sending stop signal to 2 worker threads
galaxy.jobs.runners INFO 2015-08-18 10:37:45,907 LocalRunner: Sending stop signal to 5 worker threads
galaxy.jobs.handler INFO 2015-08-18 10:37:45,907 sending stop signal to worker thread
galaxy.jobs.handler INFO 2015-08-18 10:37:45,907 job handler stop queue stopped
functional_tests.py INFO 2015-08-18 10:37:45,907 Embedded Universe application stopped
functional_tests.py INFO 2015-08-18 10:37:45,907 Cleaning up temporary files in /tmp/tmpp7gnEV/tmpS2Ul4C
galaxy.jobs.handler INFO 2015-08-18 10:37:46,876 sending stop signal to worker thread
galaxy.jobs.handler INFO 2015-08-18 10:37:46,876 job handler queue stopped
galaxy.jobs.runners INFO 2015-08-18 10:37:46,876 LWRRunner: Sending stop signal to monitor thread
galaxy.jobs.runners INFO 2015-08-18 10:37:46,876 LWRRunner: Sending stop signal to 3 worker threads
galaxy.jobs.runners INFO 2015-08-18 10:37:46,876 TaskRunner: Sending stop signal to 2 worker threads
galaxy.jobs.runners INFO 2015-08-18 10:37:46,877 LocalRunner: Sending stop signal to 5 worker threads
galaxy.jobs.handler INFO 2015-08-18 10:37:46,877 sending stop signal to worker thread
galaxy.jobs.handler INFO 2015-08-18 10:37:46,877 job handler stop queue stopped
All 1 test(s) executed passed.
hcluster_sg_parser[0]: failed
```

And tool_test_output.html is green.